utils/index.d.ts

```diff
@@ -959,6 +959,15 @@ export interface IAzureUserInput {
      * @returns A promise that resolves to the selected resources.
      */
     showOpenDialog(options: AzExtOpenDialogOptions): Promise<Uri[]>;
+
+    /**
+     * Shows a selection list of existing workspace folders to choose from.
+     *
+     * @param options Configures the behavior of the workspace folder list.
+     * @throws `UserCancelledError` if the user cancels.
+     * @returns A promise that resolves to the selected `WorkspaceFolder`.
+     */
+    showWorkspaceFolderPick(options: AzExtWorkspaceFolderPickOptions): Promise<WorkspaceFolder>;
```

Comment:
I think I like the name with picker more, but I was thinking it might be nice to keep the name mirrored with its VS Code analogue => `vscode.window.showWorkspaceFolderPick`